src/components/Widgets/Widget.style.ts

```diff
@@ -12,106 +12,115 @@ const DEFAULT_WIDGET_TOP_HOVER_OFFSET = 24;
-  width: '100%',
-  position: 'relative',
-  margin: theme.spacing(0, 'auto'),
-  zIndex: 2,
-  height: 'auto',
+  shouldForwardProp: (prop) =>
+    prop !== 'welcomeScreenClosed' && prop !== 'autoHeight',
+})<WidgetWrapperProps>(({ theme, welcomeScreenClosed, autoHeight }) => {
+  // autoHeight is used to adapt widget-height automatically instead of default 686px
+  const widgetHeight: 'auto' | number = autoHeight
+    ? 'auto'
+    : DEFAULT_WIDGET_HEIGHT;
 
-  // welcome-screen styles to be found in Widget.style.tsx + Widgets.style.tsx
-  // CSS for welcome-screen -->
-  ...(!welcomeScreenClosed && {
-    overflow: 'hidden',
-    [`@media screen and (min-height: 700px)`]: {
-      overflow: 'visible',
-    },
-  }),
-
-  '& > div:not(.alert)': {
+  return {
+    width: '100%',
     position: 'relative',
-    transitionProperty: 'margin-top',
-    transitionDuration: '.3s',
-    transitionTimingFunction: 'ease-in-out',
-    marginTop: 0,
-    maxHeight: welcomeScreenClosed ? '100%' : '50vh',
+    margin: theme.spacing(0, 'auto'),
+    zIndex: 2,
+    height: 'auto',
+
+    // welcome-screen styles to be found in Widget.style.tsx + Widgets.style.tsx
+    // CSS for welcome-screen -->
     ...(!welcomeScreenClosed && {
-      cursor: 'pointer',
-      // add margin-top to widget-wrapper when welcome-screen is closed
-      marginTop: DEFAULT_WIDGET_TOP_HOVER_OFFSET,
+      overflow: 'hidden',
+      [`@media screen and (min-height: 700px)`]: {
+        overflow: 'visible',
+      },
+    }),
 
-      '&:hover': {
+    '& > div:not(.alert)': {
+      position: 'relative',
+      transitionProperty: 'margin-top',
+      transitionDuration: '.3s',
+      transitionTimingFunction: 'ease-in-out',
+      marginTop: 0,
+      maxHeight: welcomeScreenClosed ? '100%' : '50vh',
+      ...(!welcomeScreenClosed && {
+        cursor: 'pointer',
         // add margin-top to widget-wrapper when welcome-screen is closed
-        marginTop: 0,
-      },
+        marginTop: DEFAULT_WIDGET_TOP_HOVER_OFFSET,
 
-      // positioning of widget on mobile-screens from 700px height
-      [`@media screen and (min-height: 700px)`]: {
-        marginTop: `calc( ${DEFAULT_WIDGET_TOP_OFFSET_VAR} - ${HeaderHeight.XS}px )`,
         '&:hover': {
-          marginTop: `calc( ${DEFAULT_WIDGET_TOP_OFFSET_VAR} - ${HeaderHeight.XS}px - ${DEFAULT_WIDGET_TOP_HOVER_OFFSET}px )`,
+          // add margin-top to widget-wrapper when welcome-screen is closed
+          marginTop: 0,
         },
-      },
-      // positioning of widget on mobile-screens from 900px height
-      [`@media screen and (min-height: 900px)`]: {
-        marginTop: `calc( ${DEFAULT_WIDGET_TOP_OFFSET_VAR} - ${HeaderHeight.MD}px)`,
-        '&:hover': {
-          marginTop: `calc( ${DEFAULT_WIDGET_TOP_OFFSET_VAR} - ${HeaderHeight.MD}px - ${DEFAULT_WIDGET_TOP_HOVER_OFFSET}px )`,
+
+        // positioning of widget on mobile-screens from 700px height
+        [`@media screen and (min-height: 700px)`]: {
```

Comment:
Would be good to use breakpoints value instead to avoid regresions

Reply:
This is magic voodoo happening. Please note, it´s applying styles to the `min-height` which we usually don´t have as default breakpoints. It´s needed to position the widget accordingly while the welcomeScreen is open.